pkg/utils/proxy/create.go

```diff
@@ -54,3 +55,14 @@ func BuildConfigmap(proxy Proxy) (*v1.ConfigMap, error) {
 
 	return cm, nil
 }
+
+// BuildPromService builds kubernetes prometheus ServiceMonitor CR object used for monitoring
+func BuildPromService(proxy Proxy) (*promv1.ServiceMonitor, error) {
```

Comment:
also here I would name `BuildServiceMonitor` like other functions. 
including resource name as it is, matchs with current convention.

Reply:
changed.